src/binding/c/attr_api.txt

```diff
@@ -0,0 +1,202 @@
+# vim: set ft=c:
+
+MPI_Attr_delete:
+    .desc: Deletes an attribute value associated with a key on a
+    .replace: deprecated with MPI_Comm_delete_attr
+
+MPI_Attr_get:
+    .desc: Retrieves attribute value by key
+    .skip: Fortran
+    .replace: deprecated with MPI_Comm_get_attr
+    .seealso: MPI_Attr_put, MPI_Keyval_create, MPI_Attr_delete, MPI_Comm_get_attr
+/*
+    Notes:
+        Attributes must be extracted from the same language as they were inserted
+        in with 'MPI_ATTR_PUT'.  The notes for C and Fortran below explain why.
+*/
+/*
+    Notes for C:
+        Even though the 'attribute_val' argument is declared as 'void *', it is
+        really the address of a void pointer (i.e., a 'void **').  Using
+        a 'void *', however, is more in keeping with C idiom and allows the
+        pointer to be passed without additional casts.
+
+    .N Fortran
+        The 'attribute_val' in Fortran is a pointer to a Fortran integer, not
+        a pointer to a 'void *'.
+*/
+
+MPI_Attr_put:
+    .desc: Stores attribute value associated with a key
+    .replace: deprecated with MPI_Comm_set_attr
+    .seealso: MPI_Attr_get, MPI_Keyval_create, MPI_Attr_delete, MPI_Comm_set_attr
+/*
+    Notes:
+    Values of the permanent attributes 'MPI_TAG_UB', 'MPI_HOST', 'MPI_IO',
+    'MPI_WTIME_IS_GLOBAL', 'MPI_UNIVERSE_SIZE', 'MPI_LASTUSEDCODE', and
+    'MPI_APPNUM'  may not be changed.
+
+    The type of the attribute value depends on whether C, C++, or Fortran
+    is being used.
+    In C and C++, an attribute value is a pointer ('void *'); in Fortran,
+    it is a single
+    integer (`not` a pointer, since Fortran has no pointers and there are systems
+    for which a pointer does not fit in an integer (e.g., any > 32 bit address
+    system that uses 64 bits for Fortran 'DOUBLE PRECISION').
+
+    If an attribute is already present, the delete function (specified when the
+    corresponding keyval was created) will be called.
+
```

Comment:
[not important]: Do we need a line break here? (Did you copy them manually?)

Reply:
The man page (`troff`) ignores white spaces and re-formats the presentation. Line breaks here is optional just like line breaks in code.

Reply:
I see. Okay. I believe we can fix it for a more consistent format, but I won't point it out in this review.